docs/workflows/common-workflow-steps/change-java-version.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Change Java Version
+description: Switch the current Java version to the selected one.
+tags: [change, java, version, java version, android java]
+---
+
+import Screenshot from '@site/src/components/Screenshot';
+
+# Change Java Version
+
+The **Change Java Version** step updates the JDK and Java version to the selected one during the build process.
+
+If your project requires a different JDK or Java version than the default in your runner, you can use this step to switch to the selected version. To check the default Java version used in Appcircle, refer to this document:
+
+- [Android Build Stacks - Java Version](/infrastructure/android-build-infrastructure#java-version)
+
+Note that available Java versions may vary based on the runner you are using.
+
+For self-hosted users, the versions listed for the cloud Appcircle runner may not apply. Refer to the FAQ below for guidance:
+
+- [How do I check available Java versions in the Appcircle runner?](#how-do-i-check-available-java-versions-in-the-appcircle-runner)
+
+:::tip Check Java Version for Android
+
+If you are unsure which Java version is required for your Android project, you can find it by following the steps in this FAQ:
+
+- [How can I check the Java version of my project?](#how-can-i-check-the-java-version-of-my-project)
+
+:::
+
+
+## Prerequisites
+
+There is no mandatory step for using this component.  However, if you have a step that necessitates changing the Java version (particularly build steps), the **Change Java Version** step should run before that step in the [workflow](/workflows).
```

Comment:
```suggestion
There is no mandatory step for using this component.  However, if you have a step that necessitates changing the Java version (particularly build steps), the **Select Java Version** step should run before that step in the [workflow](/workflows).
```
And additionally, please change the prerequisites introduction sentence with the new template sentence that I change in this [PR](https://github.com/appcircleio/appcircle-docusaurus/pull/731).